File: SCALA/Cap05/c05ex01.scala
// c05ex01.scala

import scala.io.StdIn._

object Main {
  def main(args: Array[String]): Unit = {
    var I: Int = 1
    var N, R: Int = 0

    while (I <= 5) {
      print("Entre um valor numerico: ")
      N = readInt()

      R = N * 3

      println("Resultado = " + R + "\n")

      I += 1
    }

    print("\nTecle <Enter> para encerrar... ")
    readLine()
  }
}

File: DART/Cap05/c05ex01.dart
// c05ex01.dart

import 'dart:io';

void main() {
  int I = 1;
  int N;
  int R;

  while (I <= 5) {
    stdout.write("Entre um valor numerico: ");
    N = int.tryParse(stdin.readLineSync() ?? '') ?? 0;

    R = N * 3;

    print("Resultado = $R\n");

    I++;
  }

  stdout.write("\nTecle <Enter> para encerrar... ");
  stdin.readLineSync();
}
